core/src/main/java/com/sap/ai/sdk/core/AiCoreService.java

```diff
@@ -43,7 +43,7 @@ public class AiCoreService implements AiCoreDestination {
   private static final String AI_RESOURCE_GROUP = "URL.headers.AI-Resource-Group";
 
   /** loads the .env file from the root of the project */
-  public static final Dotenv DOTENV = Dotenv.configure().ignoreIfMissing().load();
+  protected static final Dotenv DOTENV = Dotenv.configure().ignoreIfMissing().load();
```

Comment:
```suggestion
  private static final Dotenv DOTENV = Dotenv.configure().ignoreIfMissing().load();
```
Why not private?

Reply:
> Why not private?

`private` would also be fine for me.
I consider the possibility for users to extend/customize the class. It may be useful to them to load environment variables too. Not a strong opinion.